backend/danswer/server/documents/indexing.py

```diff
@@ -0,0 +1,23 @@
+from fastapi import APIRouter
+from fastapi import Depends
+from sqlalchemy.orm import Session
+
+from danswer.auth.users import current_admin_user
+from danswer.db.engine import get_session
+from danswer.db.index_attempt import (
+    get_index_attempt_errors,
+)
+from danswer.db.models import User
+from danswer.server.documents.models import IndexAttemptError
+
+router = APIRouter(prefix="/manage")
+
+
+@router.get("/admin/indexing_errors/{index_attempt_id}")
```

Comment:
nit: for consistency, prefer having this be `"/admin/indexing-errors/{index_attempt_id}"` (`-` instead of `_`). 

Other similar APIs (e.g. `deletion-attempt` APIs) do it this way